pasqal_cloud/__init__.py

```diff
@@ -214,6 +216,58 @@ def cancel_job(self, id: str) -> Job:
         job = Job(**job_rsp, _client=self._client)
         return job
 
+    def create_workload(
+        self,
+        workload_type: str,
+        backend: str,
+        config: Dict[str, Any],
+        wait: bool = False,
+    ) -> Workload:
+        req = {
+            "workload_type": workload_type,
+            "backend": backend,
+            "config": config,
+        }
```

Comment:
I think we could create another schema to validate the payload to create the workload.
We could ensure the backend matches one of the BackendNames, and the workload_type too.
This means `WorkloadType` and `BackendName` enums should be stored in the sdk too.

I would have a `workload` directory with a `constants.py` util for storing those.
The `CreateWorkloadSchema` could also perform some validations. For example if a backend is not available for a given workload type it could be done there.

Reply:
Both the WorkloadType and BackendName should not be disclosed in this open source directory and I thus can't make any validation on the sdk for that. When qucint will be open source we will be able to reveal (& validate) a bit more tho.